src/evrythng/tests/config.json

```diff
@@ -0,0 +1,4 @@
+{    "parameters": {
+        "API_KEY": "FSz26K5a4HlImg7hX4i1BhEE6B5JieAKDJKijzAdX2a3S8AjyF1H406qbcIxT6vHTlqe2pORLc9Ts8TJ"
```

Comment:
It's generally not a good idea to check in keys into the repo. This should be pulled in as an environment variable.